chain/telemetry/src/lib.rs

```diff
@@ -48,8 +48,9 @@ impl TelemetryActor {
             .timeout(CONNECT_TIMEOUT)
             .connector(
                 Connector::new()
-                    .conn_lifetime(Duration::from_secs(u64::max_value()))
-                    .conn_keep_alive(Duration::from_secs(30)),
+                    .conn_lifetime(Duration::from_secs(15))
+                    .conn_keep_alive(Duration::from_secs(30))
+                    .finish(),
```

Comment:
Why did we not have `.finish` before?

Reply:
Something changed in the lib, I've rebased from the older version of nearcore before lib upgrade